packages/react/ds/src/header/header.tsx

```diff
@@ -1,76 +1,105 @@
+          <ul className="gi-py-2 gi-flex gi-justify-end gi-gap-4 gi-items-center gi-h-full gi-mr-8 gi-my-1">
+            {languages.map((link, index) => (
+              <li key={index}>
+                <a
+                  href={link.href}
+                  className="focus-visible:gi-outline-offset-0 focus-visible:gi-outline-none focus-visible:gi-border focus-visible:gi-border-solid focus-visible:gi-border-yellow-400 focus:gi-border focus:gi-border-solid focus:gi-border-yellow-400 gi-block gi-text-white hover:gi-bg-black hover:gi-bg-opacity-20 gi-py-1 gi-px-2 gi-rounded-sm"
+                >
+                  {link.label}
+                </a>
+              </li>
+            ))}
+          </ul>
         </div>
-      </Container>
+      )}
+      <div
+        id="MenuContainer"
+        className="gi-h-20 gi-justify-between gi-items-center gi-flex gi-bg-emerald-800 gi-relative gi-py-3 gi-px-4 sm:gi-px-8 sm:gi-py-4"
+      >
+        <div className="gi-flex">
+          <a href={logoLink} className="xs:gi-block gi-hidden">
+            <GovieLogo />
+          </a>
+          <a href={logoLink} className="xs:gi-hidden gi-block">
+            <GovieLogoSmall />
+          </a>
+        </div>
+        <div className="gi-flex gi-items-center">
+          <ul className="gi-hidden sm:gi-flex gi-gap-4 gi-text-white">
+            {navLinks?.map((link, index) => (
+              <li key={index} className="gi-flex">
+                <a
+                  href={link.href}
+                  className="gi-text-2md gi-font-bold focus-visible:gi-outline-offset-0 focus-visible:gi-outline-none focus-visible:gi-border focus-visible:gi-border-solid focus-visible:gi-border-yellow-400 gi-font-primary gi-rounded-sm active:gi-underline active:gi-underline-offset-sm hover:gi-bg-black hover:gi-bg-opacity-20 gi-p-2 focus:gi-border focus:gi-border-solid focus:gi-border-yellow-400"
+                >
+                  {link.label}
+                </a>
+              </li>
+            ))}
+          </ul>
+          {navLinks && (
+            <div className="gi-hidden sm:gi-block gi-border-l gi-border-solid gi-border-l-white gi-h-8 gi-mx-6"></div>
+          )}
+          <label
+            htmlFor="SearchTrigger"
+            className="focus-visible:gi-outline-offset-0 focus-visible:gi-outline-none focus-visible:gi-border focus-visible:gi-border-solid focus-visible:gi-border-yellow-400 gi-hidden xs:gi-flex gi-rounded-sm hover:gi-bg-black hover:gi-bg-opacity-20 gi-p-2 gi-items-center gi-gap-md gi-cursor-pointer focus:gi-border focus:gi-border-solid focus:gi-border-yellow-400"
+          >
+            <input className="gi-hidden" id="SearchTrigger" type="checkbox" />
+            <Text className="gi-text-2md gi-font-bold gi-text-white" as="span">
+              Search
+            </Text>
+            <Icon className="search-icon gi-text-white" icon="search" />
```

Comment:
Where is defined this class `search-icon`?

Reply:
it's in the `styles.css` as a workaround for the `peer` modifier from tailwind since `peer` did not work as expected. if we can resolve the `peer` issue then we need to remove the `className` prop from the `Icon` component

Reply:
I also had to replace the `Text` component with a regular span because we needed the font-size to stay the same accross all screens and the `Text` component doesn't give us that capabilty